docs/assets/js/nodes.js

```diff
@@ -184,16 +217,38 @@ const renderWithScheduler = () => {
   scheduler.postTask(toggleMappedCategories);
 };
 
```

Comment:
A bit robust to support `scheduler` when available. Keeps things feeling snappy by putting the analytics call in the background.